tests/unit/htdocs/class/XoopsPreloadTest.php

```
@@ -0,0 +1,474 @@
+
+        $instance->triggerEvent('TEST.UPPER.CASE');
+
+        $this->assertTrue(TestEventPreload::$called, 'Uppercase event name should be normalized and matched');
+    }
+
+    /**
+     * triggerEvent() for a non-existent event does not throw or error.
+     */
+    public function testTriggerEventForNonExistentEventDoesNotError(): void
+    {
+        $instance = XoopsPreload::getInstance();
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\XoopsPreload' in method 'testTriggerEventForNonExistentEventDoesNotError'. (undefined)

(StaticAccess)